libfwupdplugin/fu-hidraw-device.c

```diff
@@ -51,7 +51,7 @@ static gboolean
 fu_hidraw_device_probe(FuDevice *device, GError **error)
 {
 	FuHidrawDevice *self = FU_HIDRAW_DEVICE(device);
-	g_autofree gchar *prop_id = NULL;
+	g_autofree gchar *prop_id = NULL, *version = NULL;
```

Comment:
I don't think this does what you think it does; I think two lines would be much clearer.